tasklist/client/src/StartProcessFromForm/index.test.tsx

```diff
@@ -137,7 +137,9 @@ describe('<StartProcessFromForm />', () => {
       screen.queryByTestId('public-form-skeleton'),
     );
 
-    fireEvent.click(
+    await user.type(screen.getByRole('textbox', {name: /is cool/i}), 'var1');
```

Comment:
@Skaiir with the latest release from `form-js` this test fails.
I believe this comes from the form register refactor, could you maybe take a quick and check if generates problems on `form-js`?

I didn't manage to reproduce it manually, but I had to add an artificial delay for the test to pass